src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenSilentParameterBuilder.cs

```diff
@@ -195,6 +195,12 @@ public AcquireTokenSilentParameterBuilder WithProofOfPossession(string nonce, Ht
             popConfig.HttpMethod = httpMethod ?? throw new ArgumentNullException(nameof(httpMethod));
             popConfig.Nonce = nonce;
 
+            //POP Auth scheme should not wrap and sign token when broker is enabled for public clients
+            if (ServiceBundle.Config.IsBrokerEnabled)
+            {
+                popConfig.SignHttpRequest = false;
```

Comment:
What about AcquireTokenInteractive and other flows? Should we put this somewhere else?

Reply:
Only ATS is shared between CCA and PCA so it needs the auth scheme. This is the only place this is needed.